src/internal_events/loki.rs

```diff
@@ -1,22 +1,13 @@
-use super::prelude::{error_stage, error_type};
 use metrics::counter;
 use vector_core::internal_event::InternalEvent;
 
 #[derive(Debug)]
-pub struct LokiEventUnlabeledError;
+pub struct LokiEventUnlabeled;
 
-impl InternalEvent for LokiEventUnlabeledError {
+impl InternalEvent for LokiEventUnlabeled {
     fn emit_metrics(&self) {
-        counter!(
-            "component_errors_total", 1,
-            "error_code" => "unlabeled",
-            "error_type" => error_type::CONDITION_FAILED,
-            "stage" => error_stage::PROCESSING,
-        );
-        counter!(
-            "processing_errors_total", 1,
-            "error_type" => "unlabeled_event",
-        );
+        counter!("processing_errors_total", 1,
+                "error_type" => "unlabeled_event");
     }
 }
```

Comment:
Looking at this again - I _think_ this is fine to actually be an error, if I remember correctly there needs to be at least one label applied for Loki to receive this. cc @jszwedko @bruceg

Reply:
I want to say the last time I debugged this sink the config didn't load if you didn't define labels 🤔

Reply:
In that case, when would that event ever actually emit?